py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
new script at the top level. this is going to eventually become the lambada function for scheduling things.